shell.nix

```diff
@@ -35,6 +34,8 @@ mkShell {
     # check the start script for problems
     shellcheck
     file
+    # Needed for rules_bazel_integration_test
+    clang
```

Comment:
Out of curiosity, why is that needed?

Reply:
This puzzled me also, ruleset implementation demanding this explicitly https://github.com/bazel-contrib/rules_bazel_integration_test/blob/main/bazel_integration_test/private/bazel_integration_test.bzl#L137-L141, but I still didn't figure out why

Reply:
Probably worth an upstream ticket. The only occurrences of `clang` in that repo according to GH search are around the `CC` variable. Maybe it's a leftover from some previous context?

Reply:
https://github.com/bazel-contrib/rules_bazel_integration_test/issues/70